### PR TITLE
Check GitHub permissions

### PR DESCRIPTION
**Pull request by rafaeljesus**
Sets `is_admin` into the jwt token when the chosen provider is Github

### CODE REVIEW

File: pkg/jwt/github/provider.go [hunk 1]
```diff
@@ -15,35 +22,70 @@ func init() {
...
 	client := NewClient()
 
-	user, err := client.CurrentUser(httpClient)
-	if err != nil {
-		return nil, err
+	var (
+		wg            sync.WaitGroup
+		user          *github.User
+		usersOrgTeams OrganizationTeams
+		errs          []error
```
Comment (italolelis) [inline pkg/jwt/github/provider.go]: `err error`

File: pkg/jwt/github/provider.go [hunk 2]
```diff
@@ -15,35 +22,70 @@ func init() {
...
+	)
+
+	wg.Add(2)
+
+	go func() {
+		defer wg.Done()
+		res, err := client.CurrentUser(httpClient)
+		if err != nil {
+			errs = append(errs, errors.Wrap(err, "failed to get github users"))
```
Comment (italolelis) [inline pkg/jwt/github/provider.go]: `err = errors.Wrap(err, "failed to get github users")`

File: pkg/jwt/github/provider.go [hunk 3]
```diff
@@ -15,35 +22,70 @@ func init() {
...
+			return
+		}
+		user = res
+	}()
+
+	go func() {
+		defer wg.Done()
+		res, err := client.Teams(httpClient)
+		if err != nil {
+			errs = append(errs, errors.Wrap(err, "failed to get github teams"))
```
Comment (italolelis) [inline pkg/jwt/github/provider.go]: `err = errors.Wrap(err, "failed to get github teams")`

File: pkg/jwt/github/provider.go [hunk 4]
```diff
@@ -15,35 +22,70 @@ func init() {
...
+			return
+		}
+		usersOrgTeams = res
+	}()
+
+	wg.Wait()
+
+	if len(errs) > 0 {
```
Comment (italolelis) [inline pkg/jwt/github/provider.go]: `if err != nil`

File: pkg/jwt/github/provider.go [hunk 5]
```diff
@@ -2,10 +2,17 @@ package github
...
 
 	"github.com/dgrijalva/jwt-go"
+	"github.com/google/go-github/github"
 	"github.com/hellofresh/janus/pkg/config"
 	"github.com/hellofresh/janus/pkg/jwt/provider"
+	"github.com/pkg/errors"
+)
+
+const (
+	janusOwners = "janus-owners"
```
Comment (italolelis) [inline pkg/jwt/github/provider.go]: This should come from an environment variable `JANUS_ADMIN_TEAM` and should be passed to the global config

**Review by italolelis — CHANGES_REQUESTED** [review]

File: pkg/jwt/github/provider.go (hunk 4 above)
Comment (rafaeljesus) [reply pkg/jwt/github/provider.go]: I think it's a bit harder to understand the code aggregating the result from 2 calls to a single variable, but not a big deal will change

**Review by italolelis — APPROVED** [review]

**Pull request merged**